oops/hosts/jwst/nircam/__init__.py

```diff
@@ -196,6 +196,12 @@ def from_hdulist(hdulist, **options):
             subfields['i_over_f'] = cal
 
         subfields['texp'] = subfields['headers'][0]['EFFEXPTM']
+        
```

Comment:
This looks like some old test code that I left behind in a previous checkin. Please delete.